source/mapping-rules/mapping-rule-options/time-series.txt

```diff
@@ -0,0 +1,122 @@
+.. _rm-timeseries:
+
+===========
+Time Series
+===========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can configure mapping rules to migrate a table that contains a date 
+time column as a :ref:`time series collections <manual-timeseries-collection>`. 
+Time series collections efficiently store time series data. In time 
+series collections, writes are organized so that data from the same 
+source is stored alongside other data points from a similar point in time.
+
+About this Task
+---------------
+
+Compared to normal collections, storing time series data in time series
+collections improves query efficiency and reduces the disk usage for
+time series data and secondary indexes.
```

Comment:
Can we add a link for "secondary indexes"? This is a new term on this page and I think it'd help to provide a link with more info.